src/app/(frontend)/api/contacts/route.ts

```diff
@@ -0,0 +1,65 @@
+import { STATUS_CODES } from '@/app/constants'
+import { NextRequest, NextResponse } from 'next/server'
+import { Resend } from 'resend'
+
+type ContactsPostReq = {
+  firstName: string
+  lastName: string
+  email: string
+}
+
+export async function POST(req: NextRequest) {
+  try {
+    const { firstName, lastName, email }: ContactsPostReq = await req.json()
+
+    if (!firstName || !lastName || !email) {
```

Comment:
nit: we could try to add some server side validation framework to do this. But we won't have too many API calls so it could be overkill. 

https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations#server-side-form-validation